02-intermediate-ruby/api-testing-vcr.md

```diff
@@ -89,8 +79,18 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
 
 Once you run a test that uses VCR, you'll notice that there will be a file in the `test/cassettes` folder with the name corresponding to the parameter provided in the `use_cassette` method.
 
-If you expect the response data to change, you must delete the cassette file.
+If you expect the response data to change, _you must delete the cassette file._
 
 Once the cassette file is created, how do we know that our tests are not still calling the API directly? One (very manual) way of doing this is by turning off your Wifi. If you ensure you have the VCR cassette files, then turn off your Wifi and run your tests again, your tests should still pass! Huzzah!
 
-**Note** You will also need to place the `VCR.use_cassette` block around your controller tests as well.  Otherwise your controller tests, which use the API will fail.
+## Wrap Up
+
+Our process for testing with VCR is as follows:
+
+1. Ensure that we are using `dotenv` to hide any personal information or secrets that we might have to send as part of a test
+
+1. Write and run our tests as normal
+
+1. Check to see that our cassettes have been instansiated.
+
+1. When a test needs to change, delete the existing cassette and proceed from step 2.
```

Comment:
If we're going to refer to a specific section do we want to use h3s so we can link to it?